src/test/java/hudson/plugins/copyartifact/CopyArtifactTest.java

```diff
@@ -346,8 +346,15 @@ public void testCopyToSlave() throws Exception {
     @Test
     public void testParameters() throws Exception {
         FreeStyleProject other = createArtifactProject(),
-                         p = createProject("$PROJSRC", null, "$BASE/*.txt", "$TARGET/bar",
+                         p = createProject("${PROJSRC}", null, "${BASE}/*.txt", "${TARGET}/bar",
                                            false, false, false, true);
+        ParameterDefinition paramDef = new StringParameterDefinition("PROJSRC",other.getName(), "");
+        ParameterDefinition paramDef2 = new StringParameterDefinition("BASE","","");
+        ParameterDefinition paramDef3 = new StringParameterDefinition("TARGET","foo","");
+        ParametersDefinitionProperty paramsDef = new ParametersDefinitionProperty(paramDef, paramDef2,paramDef3);
+        p.addProperty(paramsDef);
+        CopyArtifactPermissionProperty cpp=new CopyArtifactPermissionProperty("/"+p.getName());
```

Comment:
Why does this need to be added now—why is it in this PR? Was JENKINS-20398 somehow related?